torchprime/torch_xla_models/README.md

```diff
@@ -47,8 +47,9 @@ export LIBTPU_INIT_ARGS='--xla_tpu_scoped_vmem_limit_kib=98304 --xla_tpu_use_min
 
 tp run torchprime/torch_xla_models/train.py \
     model=llama-3-8b \
-    dataset_config_name=wikitext-103-raw-v1 \
-    global_batch_size=1024 \
+    task=basic \
+    task.global_batch_size=1024 \
+    data=wikitext \
```

Comment:
Add `data.config_name=wikitext-103-raw-v1` to switch to the larger version of wikitext dataset to keep the behavior the same.

Ditto for other applicable instances.

Reply:
oooh... i thought they are the same dataset. Will update